docs/pages/includes/enterprise/install-macos.mdx

```diff
@@ -0,0 +1,20 @@
+  You can download one of the following .pkg installers for macOS:
+
+  |Link|Binaries|
+  |-|-|
+  |[`teleport-ent-(=teleport.version=).pkg`](https://get.gravitational.com/teleport-ent-(=teleport.version=).pkg)|`teleport`<br/>`tctl`<br/>`tsh`<br/>`tbot`|
```

Comment:
We should switch from get.gravitational.com to cdn.teleport.dev here and below.

Reply:
Done.